File: modul_3/PRAK304-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main ()
{
    int nilai;
    scanf("%d", &nilai);
    if (nilai == 0){
        printf("Nol");
    }
    else if (nilai >= 1 && nilai <=9){
        printf("Satuan");
    }
    else if (nilai >= 11 && nilai <= 19){
        printf("Belasan");
    } 
    else if (nilai == 10||nilai >= 20 && nilai <=99){
        printf("Puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;  
}

File: modul_3/PRAK304-2510817220008-NamiraHumaira.py
nilai = int(input())
if nilai == 0 :
    print("Nol")
elif nilai >= 1 and nilai <= 9 :
    print("Satuan")
elif nilai >= 11 and nilai <= 19 :
    print("Belasan")
elif nilai == 10 or nilai >= 20 and nilai <= 99 :
    print("Puluhan")
else :
    print("Anda Menginput Melebihi Limit Bilangan")